doc/manuals/admin/logs.md

```diff
@@ -77,13 +77,22 @@ The different fields in each line are as follows:
     -   SUMMARY: This is a special level used by log summary traces,
         enabled with the `-logSummary` CLI option. Have a look at [the
         section on summary traces](#summary-traces) for details.
+-   **corr (correlator id)**. Can be either "N/A" (for log messages
+    "out of transaction", e.g. log lines corresponding to Orion Context
+    Broker startup), or it is a string in the UUID format.
+    An example:
+      "550e8400-e29b-41d4-a716-446655440000".
+    This 'correlator id' is either transferred from an incoming request,
+    or, if the incoming request doesn't carry any HTTP header "Fiware-Correlator",
+    the correlator is generated by the Orion context broker and then used in
+    the log file (as well as sent as HTTP header in forwarding messages and responses).
```

Comment:
As this part explains the log file I didn't put much emphasis on the how corr works, only that it's 'there' (only put a short comment within parenthesis). 
I believe the HTTP header and it's functionality should be described elsewhere ...

That said, I have no problem in including more info here also